source/PreferencesPanel.cpp

```diff
@@ -178,6 +200,30 @@ bool PreferencesPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &comma
 		if(zones[latest].Value().KeyName() != Command::MENU.KeyName())
 			Command::SetKey(zones[latest].Value(), 0);
 	}
+	else if(key == 'i' && page == 'p')
+	{
+		page = 'i';
+		if(!downloadedInfo)
+		{
+			PluginHelper::Download(PLUGIN_LIST_URL, Files::Config() + "plugins.json");
+			ifstream pluginlistFile(Files::Config() + "plugins.json");
+			pluginInstallList = nlohmann::json::parse(pluginlistFile);
+			pluginInstallPages = ((pluginInstallList.size() - (pluginInstallList.size() % MAX_PLUGIN_INSTALLS_PER_PAGE))
+				/ MAX_PLUGIN_INSTALLS_PER_PAGE) + (pluginInstallList.size() % MAX_PLUGIN_INSTALLS_PER_PAGE > 0);
+			downloadedInfo = true;
+		}
+	}
+	else if(key == 'i' && page == 'i' && selectedPluginInstall.url.size() && !selectedPluginInstall.installed)
+		installFeedbacks.emplace_back(Plugins::Install(selectedPluginInstall));
+	else if(key == 'i' && page == 'i' && selectedPluginInstall.url.size() && selectedPluginInstall.outdated)
+		installFeedbacks.emplace_back(Plugins::Update(selectedPluginInstall));
```

Comment:
It should probably be possible to update a plugin from the list without having to go look for it in the plugin index list.

Reply:
not sure if I get what you mean

Reply:
I need to go on the install page to update a plugin, instead of doing it in the list of installed plugins I have. Although in that case I'm not sure where the update button should go hmmmmmm

Reply:
but in your standard list are plugins that are not from the index list, also I think it would be better to have all online related stuff in one tab

Reply:
Sure I could do that... not totally convinced though, ~also where should I put the button for that???~